templates/layout-grid.php

```diff
@@ -9,7 +9,7 @@
     ?>
       <li>
         <figure class="gallerya__image">
-          <a href="<?= wp_get_attachment_image_src($image->ID, apply_filters('gallerya/image_size_lightbox', 'large'))[0] ?>" <?= !empty($caption) ? 'data-sub-html=".gallerya__image__caption"' : '' ?>>
+          <a href="<?= wp_get_attachment_image_src($image->ID, apply_filters('gallerya/image_size_lightbox', 'large'))[0] ?>" <?= !empty($caption) ? 'data-sub-html="' . $image->post_excerpt . '"' : '' ?>>
```

Comment:
This does not work as needed.

If `$image->post_excerpt` is empty, either the alt attribute or nothing is displayed. The `$caption` variable is generated out of `$image->post_excerpt`, so `$caption` may also be empty (same result as passing `$image->post_excerpt` but with the ability of filter the outcome.

Therefore either you should either pass `$caption` instead of the div to the data parameter, or simply remove the if condition to have an empty caption div. 

IIRC I passed the div as then it is possible to html markup in your caption. Otherwise everything is a one liner. – But you could check against.